src/sagemaker/workflow/utilities.py

```diff
@@ -41,6 +41,13 @@
     "if desired."
 )
 
+JOB_KEY_NONE_WARN_MSG_TEMPLATE = (
+    "The provided job field [{}] has not been specified. Please refer to the AWS Docs to "
+    "identify which field should be set to enable the custom-prefixing feature for jobs "
+    "created via a pipeline execution. "
+    "https://docs.aws.amazon.com/sagemaker/latest/dg/build-and-manage-access.html"
```

Comment:
Is this the correct doc?
It's pointing to "IAM Access Management"

Reply:
Yea its under the subsection. `Customize access management for SageMaker Pipelines jobs`